aten/src/ATen/Half.h

```diff
@@ -57,6 +57,10 @@ template<> AT_API double convert(Half f);
 template<> AT_API Half convert(int64_t f);
 template<> AT_API int64_t convert(Half f);
 
+Half::operator double() {
```

Comment:
is this included downstream in an two cpp files of an extension via `#include <ATen.h>`? If so, there will possibly linking errors.

Reply:
Yep thanks